src/scss/kbButtons.scss

```diff
@@ -0,0 +1,105 @@
+.keyboard-button {
+  font-size: 25px;
+  width: 60px;
+  height: 60px;
+  margin: 3px;
+  border: 2px solid black;
+  border-radius: 5px;
+  background-color: white;
+  outline: none;
+
+  &:hover {
+    background-color: #ffff99;
+  }
+
+  &:active {
+    background-color: #ffff66;
+  }
+}
```

Comment:
usually we set up all colors inside globals as well, and use variables here

Reply:
Done

Reply:
Done